src/sentry/testutils/fixtures.py

```diff
@@ -508,9 +509,6 @@ def create_event(self, event_id=None, normalize=True, **kwargs):
             kwargs['data'] = manager.get_data()
             kwargs['message'] = manager.get_search_message()
 
-        else:
-            assert 'message' not in kwargs, 'do not pass message this way'
```

Comment:
@mitsuhiko this didnt actually work, message was always passed this way because there was a setdefault above

Reply:
Might be worth checking on the search_message PR what happens there.